lib/portfinder.js

```diff
@@ -286,7 +286,7 @@ exports.getPorts = function (count, options, callback) {
       });
     });
   } else {
-    return internals.getPorts(count, options, callback);
+    internals.getPorts(count, options, callback);
```

Comment:
This and the change to `internals.getPort` above are no-ops in terms of what they do (as the functions don't return anything), but I feel this is more "semantically correct" in that we don't expect this function to return anything if not a promise.

Reply:
yeah makes sense